### ISSUE TITLE
Feature Request - Alert Time out

### ISSUE THREAD

**Issue by JDtheEE** [Mason10198/SkywarnPlus#32]
General Idea: After a set time period (24 hours, etc., maybe user settable?) time out the alert and revert all tones back to normal.

The idea being that after a storm, there is a possibility of the repeater loosing internet. Should the repeater owner (i.e. Chris ) be out of town and can't take the alert down with DTMF, the time out will ensure the repeater reverts to normal. This will keep the listeners from being confused that the repeater will send an alert even though the alert (presumably) has already passed.

**Comment by Mason10198** [comment]
I think I have devised a robust solution for this. Rather than a "timeout", just use the estimated "end" time of the alert encoded by the NWS from the last time the alert data was downloaded from the API.
- If API is unreachable, then fall back to the data stored from the last successful API call
- Run the "stored" data through a filter and remove anything with `current_time > alert_end_time`

This would have the effect of treating any "expired" alerts the same as if they were "cleared". Thus, if your courtesy tone was changed due to the existence of a Tornado Warning, and the internet goes down during the warning, then SWP would "clear" the alert on its own once the estimated end time has passed, and courtesy tones would automatically revert to normal.
